### PR TITLE
(fix)[Lock] Fixing the lock message and remove confirmation for lock.

### PR DESCRIPTION
**Pull request by mdhaman**
The problem was in the multi session the lock message displayed was incorrectly and state of lock was not maintained.

### CODE REVIEW

File: app/scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -549,7 +558,16 @@
         };
 
         $scope.lock = function() {
-            superdesk.intent('author', 'article', item);
+            var path = $location.path();
```
Comment (mdhaman) [inline app/scripts/superdesk-authoring/authoring.js]: Is there a better way then using $location?

Comment (amagdas) [inline app/scripts/superdesk-authoring/authoring.js]: The lock button should be visible only when the path is like `/authoring/:id/view`, I don't think that the check for location is needed...

Comment (mdhaman) [inline app/scripts/superdesk-authoring/authoring.js]: We need lock button in following two cases:

- when the user A opens the item that is already locked i.e /authoring/:id path. the user A would see lock message and "unlock" button. The user B who owns the lock closes the item. On closing the story, lock message on the User A screen should go away and present him with the lock button.

- User A is on the edit screen i.e. /authoring/:id and the User B unlock the story from the content list page - RHS preview pane.

Comment (amagdas) [inline app/scripts/superdesk-authoring/authoring.js]: Then I think we need to navigate the user to `/authoring/:id/view` to be consistent.

**Comment by amagdas** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**